tests/callbacks/test_tqdm_progress_bar.py

```diff
@@ -172,13 +172,8 @@ class CurrentProgressBar(TQDMProgressBar):
         val_batches_seen = 0
         test_batches_seen = 0
 
-        def on_train_batch_start(self, trainer, pl_module, batch, batch_idx):
-            super().on_train_batch_start(trainer, pl_module, batch, batch_idx)
-            assert self.train_batch_idx == trainer.fit_loop.batch_idx
```

Comment:
Both of these come from progress tracking now, so no need to test the equality. This is supposed to test the functionality of the progress bar.